src/layout/Footer.tsx

```diff
@@ -1 +1,46 @@
-export const Footer = () => <div className="font-bold">Footer</div>;
+import { FooterLink } from './components/NavLinks';
+import { SocialIcon } from './components/SocialIcon';
+import { faFacebook, faInstagram } from '@fortawesome/free-brands-svg-icons';
+import Logo from '../images/VilaMarina.png';
+
+export const Footer = () => {
+   const currentYear = new Date().getFullYear();
+
+   return (
+      <div className="footer bg-zinc-800">
+         <div className="container mx-auto p-10">
+            <div className="content flex flex-col justify-center align-middle text-center gap-7">
+               <div className="logo flex justify-center">
+                  <a href="https://www.instagram.com/vilamarina_jahorina/">
+                     <img src={Logo} alt="Vila Marina Jahorina Logo" className="w-[90px] h-[90px]" />
+                  </a>
+               </div>
+               <div className="links ">
+                  <ul className="grid grid-cols-2 justify-center gap-5 uppercase xl:flex lg:grid-cols-4">
+                     <FooterLink href="#villa-marina" text="villa marina" />
+                     <FooterLink href="#villa-jelena" text="villa jelena" />
+                     <FooterLink href="#price" text="price" />
+                     <FooterLink href="#reservation" text="reservation" />
+                     <FooterLink href="#about us" text="about us" />
+                     <FooterLink href="#gallery" text="gallery" />
+                     <FooterLink href="#location" text="location" />
+                     <FooterLink href="#contact" text="contact" />
+                  </ul>
+               </div>
+               <ul className="socials flex justify-center items-center gap-7">
```

Comment:
When you have more code like this, separate bigger code blocks with line-break. Like this:

```suggestion
               </div>
               
               <div className="links ">
                  <ul className="grid grid-cols-2 justify-center gap-5 uppercase xl:flex lg:grid-cols-4">
                     <FooterLink href="#villa-marina" text="villa marina" />
                     <FooterLink href="#villa-jelena" text="villa jelena" />
                     <FooterLink href="#price" text="price" />
                     <FooterLink href="#reservation" text="reservation" />
                     <FooterLink href="#about us" text="about us" />
                     <FooterLink href="#gallery" text="gallery" />
                     <FooterLink href="#location" text="location" />
                     <FooterLink href="#contact" text="contact" />
                  </ul>
               </div>
               
               <ul className="socials flex justify-center items-center gap-7">
```